ngraph_bridge/enable_variable_ops/ngraph_replace_op_utilities.cc

```diff
@@ -30,9 +30,8 @@ namespace ngraph_bridge {
 Status ReplaceOptimizer(Graph* graph, Node* node, Node** replacement,
                         const string replacement_node_name,
                         const string replacement_node_type,
-                        const bool just_looking,
-                        const bool outputs_ng_supported, const int graph_id,
-                        const bool is_backend_set) {
+                        const bool just_looking, const bool update_tf_tensor,
+                        const int graph_id, const bool is_backend_set) {
```

Comment:
note to self: Earlier `!outputs_ng_supported <-> needs copying`. Now that assumption has changed. In case of var feeding an encapsulate as static input, we have `outputs_ng_supported=true` but `needs copying = true` as well, thus breaking `!outputs_ng_supported <-> needs copying`.

Hence directly passing in a variable named `update_tf_tensor`